lib/src/includes/signed_video_sign.h

```diff
@@ -247,11 +247,10 @@ signed_video_get_nalu_to_prepend(signed_video_t *self,
  * bytes. Hence, the user can simply pull and prepend existing H26x NALUs. Pull NALUs to prepend
  * from signed_video_t one by one until no more generated SEIs exists, that is, when |sei_size|
  * is zero.
- * API provides the user with the NALU data as well as instructions on where to prepend that data.
  *
- * NOTE that as soon as the user pulls a new NALU to prepend, the ownership of the |nalu_data|
- * memory (see members of signed_video_nalu_to_prepend_t) is transferred. Free the |nalu_data|
- * memory with signed_video_nalu_data_free(...).
+ * NOTE: The user is responsible for allocating memory for the |nalu_data|
+ * (see members of signed_video_nalu_to_prepend_t). Ensure that you manage the memory appropriately.
```

Comment:
Do not reference an API that is going to be deprecated.
Also, when describing things it should be written anonymous, that is, no "I", "we" or "you". For example, this sentence should be written as "The user is responsible for managing the memory appropriately". This sentence though is redundant since it is already stated that the user allocates memory.